src/support/z_file_locate.erl

```diff
@@ -234,7 +235,7 @@ generate_preview(true, _Mime, Path, OriginalFile, Filters, Medium, Context) ->
             PreviewFilePath = filename:join(PreviewDir, Path),
             case z_media_preview:convert(z_convert:to_list(Filename), OriginalFile, z_convert:to_list(PreviewFilePath), Filters, Context) of
                 ok ->
-                    FileStorePath = filename:join([filename:basename(PreviewDir), Path]),
+                    FileStorePath = z_convert:to_binary(filename:join([filename:basename(PreviewDir), Path])),
```

Comment:
Line 238 is too long:                     FileStorePath = z_convert:to_binary(filename:join([filename:basename(PreviewDir), Path])),.